### ISSUE TITLE
pointPenfilterReduntantPoints is not an argument

### ISSUE THREAD

**Issue by typemytype** [typesupply/fontMath#61]
that must be `filterReduntantPoints`

https://github.com/typesupply/fontMath/blob/master/Lib/fontMath/mathGlyph.py#L291

(I can do a PR but for tonight Im to lazy, sorry :)

**Comment by typemytype** [comment]
and the pen to pointPen converter is also missing

```python
from ufoLib.pointPen import PointToSegmentPen
pointPen = PointToSegmentPen(pen)
```

**Comment by typemytype** [comment]
typo fixed in https://github.com/typesupply/fontMath/commit/43bb227cc514f12e136572d5aa6d0aec86fd3a47
point issue fixed in d1c883a53bbd79c7f199d2cec914ce43c560a2ee
